### PR TITLE
DOCSUP-11655: Document the SQLite

### PR DESCRIPTION
**Pull request by adevyatova**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #24194

### CODE REVIEW

File: docs/en/engines/database-engines/sqlite.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+toc_priority: 32
+toc_title: SQLite
+---
+
+# SQLite {#sqlite}
+
+The engine works with [SQLite](https://www.sqlite.org/index.html).
```
Comment (olgarev) [inline docs/en/engines/database-engines/sqlite.md:8]:
```suggestion
Allows to connect to [SQLite](https://www.sqlite.org/index.html) engine. 
```

File: docs/en/engines/database-engines/sqlite.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    ENGINE = SQLite('db_path')
+```
+
+**Engine Parameters**
+
+-   `db_path` — Path to SQLite file with the database.
+    
+## Data Types Support {#data_types-support}
+
+|  EngineName   | ClickHouse                                              |
```
Comment (olgarev) [inline docs/en/engines/database-engines/sqlite.md:23]:
```suggestion
|  SQLite   | ClickHouse                                              |
```

File: docs/en/engines/database-engines/sqlite.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+|---------------|---------------------------------------------------------|
+| INTEGER       | [Int32](../../sql-reference/data-types/int-uint.md)     |
+| REAL          | [Float32](../../sql-reference/data-types/float.md)      |
+| TEXT          | [String](../../sql-reference/data-types/string.md)      |
+| BLOB          | [String](../../sql-reference/data-types/string.md)      |
+
+## Specifics and recommendations {#specifics-and-recommendations}
```
Comment (olgarev) [inline docs/en/engines/database-engines/sqlite.md:30]:
```suggestion
## Specifics and Recommendations {#specifics-and-recommendations}
```

File: docs/en/engines/database-engines/sqlite.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+## Creating a Database {#creating-a-database}
+
+``` sql
+    CREATE DATABASE sqlite_database 
+    ENGINE = SQLite('db_path')
+```
+
+**Engine Parameters**
+
+-   `db_path` — Path to SQLite file with the database.
```
Comment (olgarev) [inline docs/en/engines/database-engines/sqlite.md:19]:
```suggestion
-   `db_path` — Path to a file with SQLite database.
```

File: docs/en/engines/database-engines/sqlite.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+|  EngineName   | ClickHouse                                              |
+|---------------|---------------------------------------------------------|
+| INTEGER       | [Int32](../../sql-reference/data-types/int-uint.md)     |
+| REAL          | [Float32](../../sql-reference/data-types/float.md)      |
+| TEXT          | [String](../../sql-reference/data-types/string.md)      |
+| BLOB          | [String](../../sql-reference/data-types/string.md)      |
+
+## Specifics and recommendations {#specifics-and-recommendations}
+
+SQLite stores the entire database (definitions, tables, indices, and the data itself) as a single cross-platform file on a host machine. It is locking the entire database file during writing. SQLite read operations can be multitasked, though writes can only be performed sequentially.
```
Comment (olgarev) [inline docs/en/engines/database-engines/sqlite.md:32]:
```suggestion
SQLite stores the entire database (definitions, tables, indices, and the data itself) as a single cross-platform file on a host machine. During writing SQLite locks the entire database file, therefore write operations are performed sequentially. Read operations can be multitasked.
```

File: docs/en/engines/database-engines/sqlite.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+SQLite does not require service management (such as startup scripts) or access control based on `GRANT` and passwords. Access control is handled by means of file-system permissions given to the database file itself.
+
+## Usage Example {#usage-example}
+
+Database in ClickHouse, exchanging data with the SQLite:
```
Comment (olgarev) [inline docs/en/engines/database-engines/sqlite.md:37]:
```suggestion
Database in ClickHouse, connected to the SQLite:
```

File: docs/en/engines/table-engines/integrations/sqlite.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
+---
+toc_priority: 7
+toc_title: SQLite
+---
+
+# SQLite {#sqlite}
+
+The engine provides to import and export data to SQLite and query SQLite tables directly in ClickHouse.
```
Comment (olgarev) [inline docs/en/engines/table-engines/integrations/sqlite.md:8]:
```suggestion
The engine allows to import and export data to SQLite and supports queries to SQLite tables directly from ClickHouse.
```

File: docs/en/engines/table-engines/integrations/sqlite.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+    (
+        name1 [type1], 
+        name2 [type2], ...
+    ) ENGINE = SQLite('db_path', 'table')
+```
+
+**Engine Parameters**
+
+-   `name1, name2, ...` — The column names.
+-   `type1, type2, ...` — The column types.
```
Comment (olgarev) [inline docs/en/engines/table-engines/integrations/sqlite.md:23]:
```suggestion
-   `name1`, `name2`, ... — The column names.
-   `type1`, `type2`, ... — The column types.
```

File: docs/en/engines/table-engines/integrations/sqlite.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+-   `db_path` — Path to SQLite file with the database.
```
Comment (olgarev) [inline docs/en/engines/table-engines/integrations/sqlite.md:24]:
```suggestion
-   `db_path` — Path to SQLite file with a database.
```

File: docs/en/engines/table-engines/integrations/sqlite.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+-   `table` — The SQLite table name.
```
Comment (olgarev) [inline docs/en/engines/table-engines/integrations/sqlite.md:25]:
```suggestion
-   `table` — Name of a table in the SQLite database.
```

File: docs/en/engines/table-engines/integrations/sqlite.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+## Usage Example {#usage-example}
+
+Show query creating the SQLite table:
```
Comment (olgarev) [inline docs/en/engines/table-engines/integrations/sqlite.md:29]:
```suggestion
Shows a query creating the SQLite table:
```

File: docs/en/engines/table-engines/integrations/sqlite.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+```text
+┌─col1─┬─col2──┐
+│    1 │ text1 │
+│    2 │ text2 │
+│    3 │ text3 │
+└──────┴───────┘
+```
+
+**See Also**
+
```
Comment (olgarev) [inline docs/en/engines/table-engines/integrations/sqlite.md:59]: - [SQLite](../../../engines/database-engines/sqlite.md) engine

File: docs/en/engines/table-engines/integrations/sqlite.md [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+-   [sqlite](../../../sql-reference/table-functions/sqlite.md) table function
```
Comment (olgarev) [inline docs/en/engines/table-engines/integrations/sqlite.md:60]:
```suggestion
-   [SQLite](../../../engines/database-engines/sqlite.md) engine
-   [sqlite](../../../sql-reference/table-functions/sqlite.md) table function
```

File: docs/en/sql-reference/table-functions/sqlite.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
+---
+toc_priority: 55
+toc_title: sqlite
+---
+
+## sqlite {#sqlite}
+
+Allows to performed queries on data that is stored in the `SQLite` database.
```
Comment (olgarev) [inline docs/en/sql-reference/table-functions/sqlite.md:8]:
```suggestion
Allows to perform queries on a data stored in an [SQLite](../../engines/database-engines/sqlite.md) database.
```

File: docs/en/sql-reference/table-functions/sqlite.md [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+**Syntax** 
+
+``` sql
+    sqlite('db_path', 'table_name')
+```
+
+**Arguments** 
+
+-   `db_path` — Path to SQLite file with the database. [String](../../sql-reference/data-types/string.md).
+-   `table_name` — The SQLite table name. [String](../../sql-reference/data-types/string.md).
```
Comment (olgarev) [inline docs/en/sql-reference/table-functions/sqlite.md:19]:
```suggestion
-   `db_path` — Path to a file with an SQLite database. [String](../../sql-reference/data-types/string.md).
-   `table_name` — Name of a table in the SQLite database. [String](../../sql-reference/data-types/string.md).
```

File: docs/en/sql-reference/table-functions/sqlite.md [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+
+**Returned value**
+
+-   A table object with the same columns as the original `SQLite` table.
```
Comment (olgarev) [inline docs/en/sql-reference/table-functions/sqlite.md:23]:
```suggestion
-   A table object with the same columns as in the original `SQLite` table.
```

**Comment by adevyatova** [comment]
@kssenii Hi! Can you review this PR?

File: docs/en/engines/database-engines/sqlite.md [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+SQLite stores the entire database (definitions, tables, indices, and the data itself) as a single cross-platform file on a host machine. During writing SQLite locks the entire database file, therefore write operations are performed sequentially. Read operations can be multitasked.
+SQLite does not require service management (such as startup scripts) or access control based on `GRANT` and passwords. Access control is handled by means of file-system permissions given to the database file itself.
+
+## Usage Example {#usage-example}
+
+Database in ClickHouse, connected to the SQLite:
+
+``` sql
+CREATE DATABASE sqlite_db ENGINE = SQLite('sqlite.db');
+SHOW TABLES FROM sqlite_db;
```
Comment (kssenii) [inline docs/en/engines/database-engines/sqlite.md:41]:
```suggestion
SHOW TABLES FROM sqlite_db;
```
```suggestion
CREATE DATABASE sqlite_db ENGINE = SQLite('sqlite.db');
SHOW TABLES FROM sqlite_db;
SELECT * FROM sqlite_db.table1;
INSERT INTO sqlite_db.table1 SELECT * FROM clickhouse_table;
```

**Review by kssenii — APPROVED** [review]

**Pull request merged**